src/python/pants/core/goals/typecheck_test.py

```diff
@@ -20,7 +20,6 @@
 from pants.engine.target import FieldSet, Sources, Target, Targets
```

Comment:
These could have been unit tests the whole time. We don't use `RuleRunner` because there's no reason to.